pipeline_dp/pipeline_operations.py

```diff
@@ -160,6 +174,19 @@ def sample_fixed_per_key(self, col, n: int, stage_name: str):
     def count_per_element(self, col, stage_name: str):
         return col | stage_name >> combiners.Count.PerElement()
 
+    def reduce_accumulators_per_key(self, col, stage_name: str = None):
+
```

Comment:
Please add 
TODO: use merge function from the accumulator framework.

Reply:
Done.